string name
string location
string destination
int8 points
bool success
time start_time
time end_time
